changelog/20.0/20.0.0/summary.md

```diff
@@ -335,6 +338,19 @@ The new flag `--querylog-sample-rate float` adds support for sampling queries ba
 
 The new flag `--tablet-filter-tags StringMap` adds support to VTGate for filtering tablets by tablet tag key/values, specified as comma-separated list of key:values. The tags of a tablet are defined by the VTTablet flag `--init_tags`, which is also defined as a comma-separated list of key:values.
 
+### <a id="vreplication"/>VReplication
+
+#### <a id="multi-tenant"/> Multi-tenant migrations
+
+Support for multi-tenant imports has been added to `MoveTables`. If you have a multi-tenant architecture where each
+tenant has their own database, you can import the tenants using multiple `MoveTables` workfows, one per tenant.
+Each import is initiated with the new `--tenant-id` flag. The column name (and data type) need to be specified in
+the VSchema of the target keyspace.
```

Comment:
Just a side note that we should add something in the docs that includes an example usage of the flag:
```
--tenant-id string                   (EXPERIMENTAL: Multi-tenant migrations only) The tenant ID to use for the MoveTables workflow into a multi-tenant keyspace.
```

As it's not entirely clear in the flag output that this is supposed to be a column reference from the vschema. Perhaps best to just document that in the flag's help output directly.